frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -13,6 +17,16 @@ type MenuItem = {
 const Sidebar: FunctionComponent = ({ children }) => {
   const { pathname } = useLocation();
   const [selected, setSelected] = useState(pathname);
+  const [profile, setProfile] = useState('');
+  const aContext = useContext(AuthContext);
+  const rContext = useContext(ReqContext);
```

Comment:
I would suggest changing the name from `aContext` to `authContext` so that it's more readable. Same for `rContext` -> `reqContext`.